client/src/components/Nav/index.jsx

```diff
@@ -33,10 +34,25 @@ class Nav extends Component {
                  </React.Fragment>)
   }
 
+  renderNavRecover() {
+    return (
+      <React.Fragment>
+        {
+        <NavItem><Link to={"/recover"}>forgot password?</Link>
+        </NavItem>
```

Comment:
When you add a link like this, it raises warning `<a> cannot appear as a descendant of <a>`. Better to do it this way:
`<li><Link to={"/recover"}>Forgot password?</Link></li>`

Reply:
change NavItem on li?

Reply:
Yes